saleor/graphql/checkout/tests/test_checkout_complete.py

```diff
@@ -1364,3 +1364,54 @@ def test_checkout_complete_0_total_value(
     assert not Checkout.objects.filter(
         pk=checkout.pk
     ).exists(), "Checkout should have been deleted"
+
+
+@pytest.mark.integration
+def test_checkout_complete_0_total_value_no_payment(
```

Comment:
Could we create a test for checkouts with 0 total caused by:
- voucher
- gift card
- Sale (shipping cost 0 or no shipping required)